add pam-bw conveter

the-infinity opened the pull request:
Adds "Baden-Württemberg: Park und Mitfahren" converter

AbdullahiFatola commented on src/parkapi_sources/converters/pam_bw/converter.py:
```diff
@@ -0,0 +1,77 @@
+from validataclass.validators import DataclassValidator
+
+from parkapi_sources.converters.base_converter.push import XlsxConverter
+from parkapi_sources.converters.neckarsulm.models import NeckarsulmRowInput
```
Do we still need Neckarsulm Model?

the-infinity replied:
Nope, copy paste leftover. Removed.

AbdullahiFatola commented on src/parkapi_sources/converters/pam_bw/converter.py:
```diff
@@ -0,0 +1,77 @@
+
+
+class PamBwPushConverter(XlsxConverter):
+    neckarsulm_row_validator = DataclassValidator(NeckarsulmRowInput)
```
Neckarsulm row validator is also here. I guess its a mistake?

the-infinity replied:
Yep, copy paste leftover. Removed.

derhuerst reviewed (commented):
Can you explain what "PAM" means, e.g. in a comment on the converter's class?

the-infinity commented:
> Can you explain what "PAM" means, e.g. in a comment on the converter's class?

German-english mixup (a = and). Changed it to German name and added it to the README.
